server/routes/activities/allocate-to-activity/handlers/beforeYouAllocate.test.ts

```diff
@@ -41,6 +41,7 @@ describe('Route Handlers - Allocate - Before you allocate', () => {
             payBand: { id: 1, alias: 'A' },
           },
           activity: {
+            activityId: 1,
             scheduleId: 1,
```

Comment:
If possible I would suggest making the schedule ids in the tests different to those of the activity.